sdk/search/Microsoft.Azure.Management.Search/tests/SessionRecords/SearchServiceTests/CanListServicesBySubscription.json

```diff
@@ -0,0 +1,349 @@
+{
+  "Entries": [
```

Comment:
In the future, please put session record changes in a separate commit to make review easier.

Reply:
Also, please put [Azure Search] at the beginning of each commit message to distinguish from other SDKs

Reply:
Thanks! Will be sure to do so in the future!